### ISSUE TITLE
Instant cloudflare ban-ip

### ISSUE THREAD

**Issue by Thomas-unique** [KotatsuApp/kotatsu-parsers#560]
### Source information

SushiScan.Net

### Steps to reproduce

Clic on Bypass cloudflare
-> you're banned
![Screenshot_2024-02-11-23-09-35-285_org koitharu kotatsu](https://github.com/KotatsuApp/kotatsu-parsers/assets/79210977/14829b8d-ea19-40cc-8b9a-10de5cc7127b)

### Kotatsu version

6.7

### Android version

14

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.

**Comment by nakrovati** [comment]
> This is still an issue as today, is there any way to fix this?

The app has a DNS over HTTPS feature, which can help.

**Comment by MrSquaare** [comment]
> The app has a DNS over HTTPS feature, which can help.

Sadly, none of the option of DNS over HTTPS help

Should this source be removed, or does it work for you?

**Comment by nakrovati** [comment]
No, it doesn't work for me either, but it's possible that the site creator has restricted access from non-French IPs. Or maybe the site has moved to sushiscans.fr
